examples/SharedMemory/PhysicsClientC_API.cpp

```diff
@@ -1342,21 +1342,28 @@ B3_SHARED_API int b3CreateCollisionShapeAddConcaveMesh(b3SharedMemoryCommandHand
 			command->m_createUserShapeArgs.m_shapes[shapeIndex].m_meshFileName[0]=0;
 			command->m_createUserShapeArgs.m_shapes[shapeIndex].m_numVertices = numVertices;
 
+			int totalUploadSizeInBytes = numVertices * sizeof(double) *3  + numIndices * sizeof(int);
+			char* data = (char*)malloc(totalUploadSizeInBytes);
```

Comment:
`std::vector` or `new`?

Reply:
Yes, let's use new instead of malloc here. I was just editing pybullet.c which is C only.